Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -160,7 +160,6 @@ def validate_response(client, url, method='GET'):
             raise ValueError('Internal server error occurred, please try again later')
         if raw_response[0].status == 429:
             demisto.debug('API Rate limit exceeded. Retrying in {} seconds...'.format(1))
-            time.sleep(1)
```

Comment:
we should return an error in this case, but as it is a recent contrib, let's keep it as is. from now on with the linter we will not allow this.